interface/wx/webview.h

```diff
@@ -459,12 +459,50 @@ class wxWebView : public wxControl
     */
     virtual void Reload(wxWebViewReloadFlags flags = wxWEBVIEW_RELOAD_DEFAULT) = 0;
 
-    /**
-        Runs the given javascript code.
-        @note When using wxWEBVIEW_BACKEND_IE you must wait for the current
+	/**
+        Gets current IE emulation level for the current program.
+        @return Current wxWebViewIEEmulationLevel.
+        @note Only avaliable on wxWEBVIEW_BACKEND_IE.
+	*/
+	virtual wxWebViewIEEmulationLevel GetEmulationLevel() = 0;
+
+	/**
+        Sets IE emulation level.
+        @param level A level from wxWebViewIEEmulationLevel enum.
+        @return True if level can be set, false if it is not.
+        @note Only avaliable on wxWEBVIEW_BACKEND_IE.
+	*/
+    virtual bool SetEmulationLevel(wxWebViewIEEmulationLevel level) = 0;
+
+    /**
+        Runs the given JavaScript. Also it does not return the result, it returns 
```

Comment:
"Also" is out of place here. Just say that it returns true if the script was run successfully and additionally returns the script return value in `output`.